src/sentry/api/serializers/models/dashboard.py

```diff
@@ -41,6 +41,7 @@ class DashboardWidgetQueryResponse(TypedDict):
     widgetId: str
     onDemand: list[OnDemandResponse]
     isHidden: bool
+    selectedAggregate: int
```

Comment:
Actually I just realized we need to update the API docs for this change. Since `selectedAggregate` may not always be set, let's just redefine this so it can be `null` on the frontend. You'll want to change it to `Optional[int]`, check out the code snippet here: https://develop.sentry.dev/application/api/public/#success-responses. Optional needs to be imported like `from typing import Optional`

You'll also need to update the API examples in `src/sentry/apidocs/examples/dashboard_examples.py` so the widget queries have `selectedAggregate` set. It's fine if we just set them to `None`.